### ISSUE TITLE
Star Wars robots

### ISSUE THREAD

**Issue by Floppy** [peterkwells/the-socioeconomic-class-of-robots-in-films#1]
I disagree with the classification of R2D2 and C3PO. OK, yeah, threepio *sounds* middle class, but droids in the Star Wars universe appear to have no rights, no recognition, no respect. Droids are basically intelligent machines that are treated like kitchen appliances by all the humans, even those on the "good" side. I'd definitely put them more in a "slavery" class, myself.

**Comment by peterkwells** [comment]
I think that goes for most robots in this particular historic story but R2D2 and C3P0 seem to be emancipated, Luke in particular gives them freedom. C3P0 seems like a reasonably high-class butler and free to wander even if he keeps going back to his "master".

**Comment by Floppy** [comment]
But do they get a medal at the end of ANH? Do they bollocks.

**Comment by Floppy** [comment]
#medalsfordroids

**Comment by peterkwells** [comment]
+1 to #medalsfordroids but Chewie didn't get no medal and is clearly no slave ([Wookiepedia](http://starwars.wikia.com/wiki/Chewbacca)). I think the lack of medals suggests caste structures or racism/speciesm rather than the type of[ informal class definitions we see in the UK](https://en.wikipedia.org/wiki/Social_structure_of_the_United_Kingdom#Informal_classifications_and_stereotypes).

**Comment by Floppy** [comment]
Yes, caste structure makes more sense. They're definitely second or third class citizens. If they are even citizens at all. #votesfordroids
